Automattic/wp-calypso
# Signup: add ability to test new Page Builder experience

#### Changes proposed in this Pull Request

Add support for the upcoming Page Builder abtest

**Note**: This is to allow us to test this more broadly internally. It will currently only work in dev and staging environments. The functions in `lib/signup/page-builder` are annotated with how the `abtest` functionality will work once we get to that point.

#### Testing instructions

* Apply D25718-code to your sandbox
* Sandbox `public-api.wordpress.com` and the site you are going to create in signup (eg `mycooltestingsite.wordpress.com`
* Go to `http://calypso.localhost:3000/start/onboarding` and choose Business in the first step
* Complete signup
* Verify that you land on `/block-editor/page/YOURSITE/2` and that you see a functional Site Header block a the top of the editor like this:
  ![image](https://user-images.githubusercontent.com/195089/55116506-742e3a00-50b5-11e9-8506-9b41410b2bc2.png)
* Try signing up with the other flows and ensure that the Page Builder experience is _not_ loaded.
